Makefile-ponyc

```diff
@@ -382,7 +392,7 @@ ifneq ($(MAKECMDGOALS),clean)
   llvm.include = $(shell $(loopit))
   #$(info llvm.include="$(llvm.include)")
 
-  llvm.libs    := $(shell $(LLVM_CONFIG) --libs $(LLVM_LINK_STATIC)) -lz -lncurses
```

Comment:
this needs to be made build target dependent so it doesnt break macos (and maybe others) yes?

Reply:
It is target dependent. That's what `llvm-config --system-libs` fills in.

Reply:
O nice. I learned a thing. That is wicked cool.